src/Game.js

```diff
@@ -40,6 +40,7 @@ export class Game {
 
   tick(delta) {
     this.update(delta);
+    this.logic.checkCollisions(this, this.entities);
```

Comment:
Can you move this into the `update` method just before where we remove "dead" entities instead? We'll want to be able to remove entities after a collision has occurred.